addons/website/views/snippets/s_product_list.xml

```diff
@@ -2,55 +2,56 @@
 <odoo>
 
 <template name="Items" id="s_product_list">
+    <t t-set="url" t-value="'http://www.%s.odoo.com/'% request.db"/>
```

Comment:
@oussamamessaoudi You should use https and not add www

@sbuhl Not convince by the change though... you prefer to reach a 404 with a random db name that reaching odoo.com. Even an empty link would be better IMO